test/decl/var/properties.swift

```diff
@@ -1347,9 +1347,9 @@ class LazyPropInClass {
 // https://github.com/apple/swift/issues/57936
 
 enum E1_57936 {
-  var foo: Int {} // expected-error{{missing return in accessor expected to return 'Int'}}
+  var foo: Int {} // missing return expectations moved to `SILOptimizer/missing_returns`
 }
 
 enum E2_57936<T> {
-  var foo: T {} // expected-error{{missing return in accessor expected to return 'T'}}
+  var foo: T {} // missing return expectations moved to `SILOptimizer/missing_returns`
 }
```

Comment:
These tests were only added to check the diagnostic, so I think we can remove them here in favor of the ones in the SILOptimizer test